tcs-service/src/main/java/com/transformuk/hee/tis/tcs/service/api/validation/CurriculumMembershipValidator.java

```diff
@@ -93,6 +97,32 @@ public void validate(CurriculumMembershipDTO cmDto)
     }
   }
 
+  /**
+   * Validate a CurriculumMembershipDTO.
+   * for bulk update
+   *
+   * @param cmDto the curriculumMembership Dto to validate
+   */
+  @Transactional(readOnly = true)
+  public void validateForBulkUploadPatch(CurriculumMembershipDTO cmDto)
+      throws MethodArgumentNotValidException, NoSuchMethodException {
+    List<FieldError> fieldErrors = new ArrayList<>();
+    if (cmDto.getCurriculumEndDate() != null && cmDto.getCurriculumStartDate() != null) {
```

Comment:
If curriculumStartDate or curriculumEndDate in this dto is still null, or both are null, how do you handle them?
And I think we also need tests for them.

Reply:
I will remove it